### DISCUSSION TITLE
What if I run out of credits?

### DISCUSSION THREAD

**Discussion by osackett** [EY-Data-Science-Program/2021-Better-Working-World-Data-Challenge#46] [2.0 Getting your free Azure credits]
Are you running low on credits? 

Did you know you can access Azure for Students credits through the EY Data Challenge Platform? These credits are available even if you've already used Azure for Students before!

To request a voucher, please go to the [EY DataScience platform](https://datascience.ey.com/), enroll in "Challenge 1: Fire Mapping", click the "Request Voucher" button and follow the instructions.

We will also be giving away bonus Azure credits throughout the Challenge, so watch for announcements in weekly newsletters!

**Comment by patricktsanght** [comment]
Hello, I have an Azure for Students account before. I am following the steps but I don't think the "Start Free" button would lead me to a page that I can redeem my code.
Would you advise how I can get those credits?
Many thanks!!

**Reply by EY-glowglowglow** [reply]
Hi there, you will still be able to request for a voucher if you are an eligible student. :)
Please request for your voucher on the [challenge platform](https://datascience.ey.com/).

**Reply by patricktsanght** [reply]
Hi, thank you so much, and I do have the voucher. But how do I redeem the voucher? I followed the instruction, but it just took me to my existing account, without using the voucher.

**Reply by EY-glowglowglow** [reply]
Hi there, you'll need to create a new account/subscription to use the voucher @RynageP . There isn't a way to top-up an existing account for Azure for Students.
